tests/test_codegen_directive.cpp

```
@@ -408,6 +408,17 @@ TEST_F(DirectiveTest, NativeLibraryExtraArgsError) {
     }, std::runtime_error);
 }
 
+// 10e-1. @native(123) — non-string positional argument causes validation error
+TEST_F(DirectiveTest, NativeNonStringArgError) {
+    try {
+        runSource("@native(123)\nfunction foo(x: int) -> int\n");
+        FAIL() << "Expected std::runtime_error";
+    } catch (const std::runtime_error &e) {
```

Cppcheck (2.20.0)

[style] The function 'isFunctionTypeName' is never used.

(unusedFunction)